jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -137,7 +180,7 @@ public final void run() {
 	 * @return {@link true} or {@link false} based on success/failure
 	 */
 	public static boolean yield(ContinuationScope scope) {
-		throw new UnsupportedOperationException();
+		return yieldImpl(scope);
```

Comment:
id like to see us use scope as a param before passing it to the VM. Personally I dont think we'll need it in the VM